articles/20240819_dev_con_vs_traditional_dev.md

```diff
@@ -0,0 +1,67 @@
+### Dev Containers vs. Traditional Development Environments: Pros and Cons
+
+The debate between traditional development setups and containerized environments has gained momentum, especially as the latter becomes more popular. This article delves into the pros and cons of Dev Containers versus traditional development environments, helping developers make an informed decision.
+
+#### What Are Dev Containers?
+
+Dev Containers are development environments packaged within containers, offering a consistent and isolated setup that runs across various platforms. Powered by tools like Docker and orchestrated through integrated development environments (IDEs) like Visual Studio Code (VSCode), these containers encapsulate all dependencies, tools, and configurations needed for a project.
+
+#### Traditional Development Environments
+
+Traditional development environments involve setting up the necessary tools, libraries, and dependencies directly on a developer’s machine. This approach has been the norm for decades, where developers install and configure everything from the operating system to the programming language and third-party libraries manually.
+
```

Comment:
It would be great to have a simple illustration for comparison, it could be done with excalidraw or tldraw.